atlascope/core/models/detected_nucleus.py

```diff
@@ -112,12 +118,29 @@ class DetectedNucleus(models.Model):
     )
 
 
+def similar_nuclei(nucleus, queryset=None):
+    """Return a queryset of nuclei ordered by similarity."""
+    if queryset is None:
+        queryset = DetectedNucleus.objects.all()
+    return queryset.annotate(
+        dissimilarity=CubeDistance('fingerprint', nucleus.fingerprint)
+    ).order_by('dissimilarity')
```

Comment:
Does this order from least dissimilar to most dissimilar?

Reply:
Yes. This orders in ascending order the "dissimilarity" measure. An image is identical if `dissimilarity == 0`. Higher values of "dissimilarity" mean the images differ more.